controller/static/index.html

```diff
@@ -248,7 +248,25 @@ <h2 class="accordion-header" id="headingBotCreate">
                                     <label for="newBotWalletExported" class="form-label">Exported, i.e. `lotus wallet export`: (optional)</label>
                                     <input class="form-control" type="text" id="newBotWalletExported" value="">
                                 </div>
-
+                                <div style="padding: 20px; background: #EEE; margin: 20px 0;">
+                                <h4>Deployment Parameters</h3>
+                                <div id="botHelmChartVersion">
+                                    <label for="newBotHelmChartVersion" class="form-label">Helm chart version of lotus-bundle. Leave blank to use latest</label>
+                                    <input class="form-control" type="text" id="newBotHelmChartVersion" value="0.0.4">
+                                </div>
+                                <div id="botHelmChartRepoUrl">
+                                    <label for="newBotHelmChartRepoUrl" class="form-label">Helm chart repo URL for lotus-bundle (optional)</label>
+                                    <input class="form-control" type="text" id="newBotHelmChartRepoUrl" value="" placeholder="https://filecoin-project.github.io/helm-charts">
+                                </div>
+                                <div id="botLotusDockerRepo">
+                                    <label for="newBotLotusDockerRepo" class="form-label">Lotus Docker Image (optional)</label>
+                                    <input class="form-control" type="text" id="newBotLotusDockerRepo" value="" placeholder="coryschwartz/lotus">
```

Comment:
Ultimately, it'd be great to move this to filecoin/lotus. coryschwartz/lotus:sidecar is based on this https://github.com/filecoin-project/lotus/pull/6544 -- just lotus with the init to download latest stateroots built in. With that merged, we can have nightly and stable releases.